config/routes.rb

```diff
@@ -45,6 +45,10 @@
 
   match '/happenstance' => 'predictions#happenstance', :as=> :happenstance
 
+  resource :credence,
+    :controller => 'credence',
+    :only => [:show, :update, :destroy]
+
```

Comment:
This should be :credence_games, not :credence, and then you wouldn't have to specify a controller.